docs/reference/mapping/fields/source-field.asciidoc

```diff
@@ -7,8 +7,7 @@ searchable), but it is stored so that it can be returned when executing
 _fetch_ requests, like <<docs-get,get>> or <<search-search,search>>.
 
 If disk usage is important to you then have a look at
-<<synthetic-source,synthetic `_source`>> which shrinks disk usage at the cost of
-only supporting a subset of mappings and slower fetches or (not recommended)
+<<synthetic-source,synthetic `_source`>> which shrinks disk usage at the cost of slower fetches or (not recommended)
```

Comment:
Maybe rename `at the cost of slower fetches or (not recommended)` into `at the cost of slower access to _source, in get and search APIs`